src/components/dashboard/ReceiveConfirmation.js

```diff
@@ -38,6 +42,22 @@ const ReceiveConfirmation = ({ screenProps, styles, ...props }: ReceiveProps) =>
     return generateShareLink('receive', code)
   }, [code])
 
+  //FIXME: RN need to restore native share after merge with master
+  //and adding the share animation
+  const shareAction = async () => {
+    if (isMobile && navigator.share) {
```

Comment:
This implementation is outdated. We already refactor it to be compatible with the react-native Share.share method.